remotespark/livyclientlib/livyclient.py

```diff
@@ -10,17 +10,18 @@ class LivyClient(object):
 
     def __init__(self, session):
         self._session = session
+        self._session.create_sql_context()
```

Comment:
How expensive is this operation? It may be worthwhile to lazily delay this until an operation requiring the SQL context is requested.